### PR TITLE
Use gofumpt for format workflow

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>
See #1278

### CODE REVIEW

**Comment by kevindiu** [comment]
I executed the command I suggested [here](https://github.com/vdaas/vald/pull/1281/files#r647291067) in #817 , and there are little difference only after these formattors runs.  I think these suggestion are good, almost LGTM 

``` 
 internal/config/server_test.go                               |  4 ++--
 internal/db/storage/blob/s3/copier/option.go                 |  4 +---
 internal/db/storage/blob/s3/deleter/option.go                |  4 +---
 internal/k8s/statefulset/statefulset_test.go                 |  7 -------
 internal/net/http/client/option_test.go                      |  4 ++--
 pkg/rebalancer/storage/controller/handler/rest/option.go     |  4 +---
 pkg/rebalancer/storage/controller/service/rebalancer.go      | 12 +++++-------
 .../storage/controller/service/rebalancer_option.go          |  4 +---
 pkg/rebalancer/storage/job/service/job/job.go                |  2 +-
 9 files changed, 14 insertions(+), 31 deletions(-)
```

File: Makefile [hunk 1]
```diff
@@ -362,12 +362,13 @@ update: \
...
 	format/yaml
 
-.PHONY: update/goimports
-## run goimports for all go files
-update/goimports:
+.PHONY: format/go
+## run gofumpt, goimports for all go files
+format/go:
+	find ./ -type d -name .git -prune -o -type f -regex '.*\.go' -print | xargs gofumpt -w
 	find ./ -type d -name .git -prune -o -type f -regex '.*\.go' -print | xargs goimports -w
```
[the remark this answers is not in the document]
Comment (rinx) [reply Makefile:372]: Thanks for your feedback.

I prefer
```suggestion
	find ./ -type d -name .git -prune -o -type f -regex '.*[^\.pb]\.go' -print | xargs gofumpt -w
	find ./ -type d -name .git -prune -o -type f -regex '.*[^\.pb]\.go' -print | xargs goimports -w
```

**Comment by hlts2** [comment]
thank you fornPR!
I think it looks good to me.
if you will fix [this](https://github.com/vdaas/vald/pull/1281#discussion_r647893717), I will approve this pr!

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevindiu) [reply Makefile:372]: yes, your solution is better

[the remark this answers is not in the document]
Comment (rinx) [reply Makefile:372]: thanks! I'm going to commit its change.

**Review by kevindiu — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**